CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Windows Support.
+
+### Changed
+
+- Completely replace keystore generation mechanism for Ethereum mainnet.
```

Comment:
Why are these two changes on the same PR, is there a reason for that?

Reply:
Windows conflict was with the fact we had to use https://github.com/creack/pty wich doesn't support it. This package was used because of the stacking deposit cli usage for keystore generation. With the new method for keystore generation a requirement for windows was to remove that dependency completely.